<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePurchasedService</class>
 <widget class="QWidget" name="ChangePurchasedService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить/Изменить услуги</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 13pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>379</width>
     <height>290</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID посетителя</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ID сотрудника</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>ID услуги</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Цена</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Время начала</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Время конца</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateTimeEdit" name="StartTime"/>
    </item>
    <item row="5" column="1">
     <widget class="QDateTimeEdit" name="EndTime"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="Visitor_ID"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="Staff_ID"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="Services_ID"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>ОК</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="Price"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
